cmd/acra-keys/keys/keystore.go

```diff
@@ -39,6 +40,39 @@ type KeyStoreParameters interface {
 	KeyDirPublic() string
 }
 
+// CommonKeyStoreParameters is a mix-in of command line parameters for key store construction.
+type CommonKeyStoreParameters struct {
+	keyDir       string
+	keyDirPublic string
+}
+
+// KeyDir returns path to key directory.
+func (p *CommonKeyStoreParameters) KeyDir() string {
+	return p.keyDir
+}
+
+// KeyDirPublic returns path to public key directory (if different from key directory).
+func (p *CommonKeyStoreParameters) KeyDirPublic() string {
+	if p.keyDirPublic == "" {
+		return p.keyDir
+	}
+	return p.keyDirPublic
+}
+
+// Register registers key store flags with the given flag set.
+func (p *CommonKeyStoreParameters) Register(flags *flag.FlagSet) {
+	p.RegisterPrefixed(flags, "", "")
+}
+
+// RegisterPrefixed registers key store flags with the given flag set, using given prefix and description.
+func (p *CommonKeyStoreParameters) RegisterPrefixed(flags *flag.FlagSet, flagPrefix, desciptionSuffix string) {
```

Comment:
```suggestion
func (p *CommonKeyStoreParameters) RegisterPrefixed(flags *flag.FlagSet, flagPrefix, descriptionSuffix string) {
```

Reply:
O, autocomplete! Misspell once – write like that everywhere.